### ISSUE TITLE
Failed to write to dbms_output (Oracle 11g)

### ISSUE THREAD

**Issue by dmlvc** [utPLSQL/utPLSQL#273]
Oracle Database 11g Enterprise Edition Release 11.2.0.3.0 - 64bit Production
NLS_CHARACTERSET **AL32UTF8** 
NLS_LANGUAGE **RUSSIAN** 
NLS_LENGTH_SEMANTICS **CHAR** 

```
SQL> call ut.run('my_schema');
call ut.run('my_schema')
     *
Ошибка в строке 1:
ORA-22813: значение операнда превышает системный лимит ORA-06512: на
"UT3.UT_OUTPUT_BUFFER", line 93
ORA-06512: на  "UT3.UT_OUTPUT_BUFFER", line 103
ORA-06512: на  "UT3.UT", line 158
ORA-06512: на  "UT3.UT", line 164
ORA-06512: на  line 1

```

I fixed it for myself by changing the return type of the function get_lines to limited one:

```
create or replace type ut_varchar2_rows as table of varchar2(4000)
...
ut_output_buffer: 
function get_lines(a_reporter_id varchar2, a_timeout_sec naturaln := gc_max_wait_sec) return ut_varchar2_rows pipelined;

```
Could you accept this fix?

**Comment by jgebal** [comment]
Hi @dmlvc, thanks for reporting this bug.

Did you only change the datatype for the function return values and nothing more?
It seems odd that this actually fixed the issue. I would expect the function get_lines to throw an error internally after this change.

You may contribute via pull-request to the existing code or we will find a way to fix this ourselves.

**Comment by dmlvc** [comment]
Hello, Jacek!

> Did you only change the datatype for the function return values and nothing more?

I also changed type of local variable _l_results_ to _ut_varchar2_rows_ so it matches to the function's return value.

I have not tested my changes much, but I don't get any errors like this any more.

I believe the problem is in internal SQL restriction on varchar2 length (max 4000 bytes) and some paranoid checks of type compatibility.

You should better fix this yourself because I don't have ability to test it properly right now.

**Comment by jgebal** [comment]
Hi @dmlvc 
Can you checkout the branch:
https://github.com/jgebal/utPLSQL/tree/feature/fix_ut_output_buffer_issues
I've addressed the issue as per your description. I'm moving a bit blind however as I cannot really reproduce the issue.
It would be great if you could verify it before I will merge it to the project codebase.

**Comment by scanbix** [comment]
I have same issue on Oracle 11.2

**Comment by jgebal** [comment]
@scanbix Can you provide full now settings (db/instance/client) as well as isolated test case/scenario? 
That will allow us to validate the issue and the fix.
At the moment we do not have environment to validate it and we don't have full understanding of the issue.

**Comment by scanbix** [comment]
Can you invite me to the slack chat?

**Comment by jgebal** [comment]
Just click on the slack-chat icon from readme doc and follow instructions.

**Comment by jgebal** [comment]
This should be now resolved in develop branch by PR #278 that was already
